src/Controller/ProductEnqueueController.php

```diff
@@ -75,13 +97,13 @@ public function enqueueAction(int $productId): Response
         foreach ($alreadyEnqueued as $code) {
             $this->addFlash(
                 'error',
-                $translator->trans('webgriffe_sylius_akeneo.ui.product_already_enqueued', ['code' => $code])
+                $this->translator->trans('webgriffe_sylius_akeneo.ui.product_already_enqueued', ['code' => $code]) // @phpstan-ignore-line
```

Comment:
The `// @phpstan-ignore-line` is necessary?

Reply:
For phpstan here the translator property can be null, but it is impossible because it's always instantiated.

Reply:
I supposed so. That's probably a bug on phpstan